src/en/clouds-maas-manual.md

```diff
@@ -0,0 +1,68 @@
+Title: Manually adding MAAS clouds
+
+# Manually adding MAAS clouds
+
+MAAS clouds may be registeres with juju using the interactive
+`add-cloud` command. If you wish to define these clouds in a YAML
+configuration file instead, this is also possible. The YAML file
+describing a MAAS cloud is straightforward - all
+that varies is the endpoint and the name, since they use the same
+authentication method. For example, below is the file maas-clouds.yaml:
+ 
```

Comment:
Consider, for brevity, although what you have is fine:
...`add-cloud` command. It is also possible to define these clouds using a YAML file containing a straightforward description. All that varies is the endpoint and the name, since they use the same authentication method. Here is an example: